package.json

```diff
@@ -1,24 +1,25 @@
 {
```

Comment:
I removed the changes induced by 'npm init', built the package and tested it again. It works OK.